templates/features/features_layout.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base_layout.html' %}
+
+{% block content %}
+<section class="p-strip--main">
+  <div class="row">
+    <div class="col-2">
+      <div class="p-side-navigation is-at-the-top" id="drawer">
+        <a href="#drawer" class="p-side-navigation__toggle js-drawer-toggle u-hide--medium u-hide--large" aria-controls="drawer">
+          Toggle side navigation
```

Comment:
This button is full with on small screens. Can you look at the differences here with Vanilla or the usage on juju.is/mission. Please.

Reply:
@anthonydillon I've made the other suggested changes, (thank you for that!) I've copied the code from Vanilla and checked it against juju/mission but can't see why it's still full width - so still working on this one.